bevy_lint/tests/ui/insert_message_resource/main.fixed

```diff
@@ -0,0 +1,37 @@
+#![feature(register_tool)]
+#![register_tool(bevy)]
+#![deny(bevy::insert_message_resource)]
+
+use bevy::prelude::*;
+
+#[derive(Message)]
+struct Foo;
+
+fn main() {
+    let mut app = App::new();
+    App::new().add_event::<Foo>();
+    //~^ ERROR: called `App::init_resource::<Messages<Foo>>()` instead of
+    // `App::add_message::<Foo>()`
+
+    App::add_event::<Foo>(&mut app);
```

Comment:
Something's weird here, these should be `add_message()` instead of `add_event()`. It may be related to all the comments I made in `insert_message_resource`.